modules/aws/lambda.go

```diff
@@ -0,0 +1,78 @@
+package aws
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/service/lambda"
+)
+
+// InvokeFunction invokes a lambda function.
+func InvokeFunction(t *testing.T, region, functionName string, payload interface{}) []byte {
+	out, err := InvokeFunctionE(t, region, functionName, payload)
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
NIT: we've been standardizing on using `require.NoError(t, err)` in place of `if => t.Fatal`. The import path is `"github.com/stretchr/testify/require"`.